src/test/java/com/hocs/test/glue/WorkstacksStepDefs.java

```diff
@@ -39,6 +40,77 @@
 
     private Page page; //more pages
 
+    Homepage homepage;
+
+    Workstacks workstacks;
+
+    @When("^I allocate the case to myself$")
+    public void allocateToMe() {
+        homepage.firstStageFindMyCase();
+        workstacks.clickAllocateToMeButton();
+        homepage.goHome();
+
+    }
+
+    @When("^I unallocate the case from myself$")
+    public void unallocateCase() {
+        homepage.selectMyCases();
+        workstacks.clickCheckboxWhereCaseReferenceIs();
+        workstacks.clickUnallocateCasesButton();
+
+    }
+
+    @When("^I select the check box against the case$")
+    public void selectCaseCheckbox() {
+        homepage.firstStageFindMyCase();
+        workstacks.clickCheckboxWhereCaseReferenceIs();
+    }
+
+
+    @Then("^The case is added to My Cases$")
+    public void assertThatCaseIsAllocatedToMe() {
```

Comment:
We are avoiding 'I' and 'me' naming conventions where possible so 'current user' might be more appropriate.